docs/src/man/basics.md

```diff
@@ -0,0 +1,1417 @@
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+```
+
+Compare this with accessing the vector contained in column `3`:
+
+```jldoctest dataframe
+julia> german[:, 3]
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+```
+
+We can get a single cell or element with the same syntax to get an element of an array:
+```jldoctest dataframe
+julia> german[4, 4]
+2
+```
+
+or to get a new `DataFrame` that is subset of rows and columns:
+```jldoctest dataframe
+julia> german[4:5, 4:5]
+2×2 DataFrame
+ Row │ Job    Housing
+     │ Int64  String
+─────┼────────────────
+   1 │     2  free
+   2 │     2  free
+```
+
+Assignment of a scalar to a data frame can be done in ranges using broadcasting:
+```jldoctest dataframe
+julia> german[:, 2] .= 4 # an in-place replacement of values stored in column number 2 by 4
+1000-element view(::Vector{Int64}, :) with eltype Int64:
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ ⋮
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+ 4
+
+julia> german
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0      4  male        2  own      NA               little          ⋯
+    2 │     1      4  female      2  own      little           moderate
+    3 │     2      4  male        1  own      little           NA
+    4 │     3      4  male        2  free     little           little
+    5 │     4      4  male        2  free     little           little          ⋯
+    6 │     5      4  male        1  free     NA               NA
+    7 │     6      4  male        2  own      quite rich       NA
+    8 │     7      4  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993      4  male        3  own      little           little          ⋯
+  995 │   994      4  male        2  own      NA               NA
+  996 │   995      4  female      1  own      little           NA
+  997 │   996      4  male        3  own      little           little
+  998 │   997      4  male        2  own      little           NA              ⋯
+  999 │   998      4  male        2  free     little           little
+ 1000 │   999      4  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+```
+
+## Not, Between, Cols, and All selectors
```

Comment:
why it is 2nd level section, and above was 3rd level?

Reply:
because these were the important and mostly used functions. So, I wanted to take it a priority.

Reply:
So what is the logic of making something `###` level section vs. `##` level section?

Reply:
Yeah, you are right. It is not making any sense. Fixing it.